cdap-ui/app/features/services/controllers/tabs/runs/tabs/status/make-request-ctrl.js

```diff
@@ -108,8 +110,12 @@ angular.module(PKG.name + '.feature.services')
       }
 
       dataSrc.request(requestObj)
-        .then(function(res) {
+        .then(function success(res) {
           vm.response = res;
+          vm.loading = false;
+        }, function error(err) {
+          vm.response = 'Some error has occured: ' + err;
```

Comment:
My thought was to just show the entire error object for ease of debugging. What do you think? @ajainarayanan